scripts/dts/edtlib.py

```diff
@@ -160,12 +161,17 @@ def __init__(self, dts, bindings_dirs, warn_file=None,
           If True, a warning is printed if a node has a 'reg' property where
           the address of the first entry does not match the unit address of the
           node
+
+        default_prop_types (default: False):
```

Comment:
Need to fix comment to match the code `(default: False) -> (default: True)`

Reply:
Thanks for catching that